src/Sylius/Bundle/ShopBundle/Grid/Account/OrderGrid.php

```diff
@@ -0,0 +1,106 @@
+
+final class OrderGrid extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public function __construct(
+        private string $resourceClass,
+        private CustomerContextInterface $customerContext,
+        private ChannelContextInterface $channelContext,
+    ) {
+    }
+
+    public static function getName(): string
+    {
+        return 'sylius_shop_account_order';
+    }
+
+    public function buildGrid(GridBuilderInterface $gridBuilder): void
+    {
+        $gridBuilder
+            ->setRepositoryMethod('createByCustomerAndChannelIdQueryBuilder', [
+                $this->customerContext->getCustomer()?->getId(),
+                $this->channelContext->getChannel()->getId(),
+            ])
+            ->orderBy('checkoutCompletedAt', 'desc')
+            ->addField(
+                TwigField::create('number', '@SyliusShop/Account/Order/Grid/Field/number.html.twig')
+                    ->setLabel('sylius.ui.number')
+                    ->setSortable(true)
+            )
+            ->addField(
+                DateTimeField::create('checkoutCompletedAt', 'm/d/Y')
+                    ->setLabel('sylius.ui.date')
+                    ->setSortable(true)
+            )
+            ->addField(
+                TwigField::create('shippingAddress', '@SyliusShop/Account/Order/Grid/Field/address.html.twig')
+                    ->setLabel('sylius.ui.ship_to')
+            )
+            ->addField(
+                TwigField::create('total', '@SyliusShop/Account/Order/Grid/Field/total.html.twig')
+                    ->setLabel('sylius.ui.total')
+                    ->setPath('.')
+                    ->setSortable(true, 'total')
+            )
+            ->addField(
+                TwigField::create('state', '@SyliusUi/Grid/Field/label.html.twig')
+                    ->setLabel('sylius.ui.state')
+                    ->setSortable(true)
+                    ->addOptions([
+                        'vars' => [
+                            'labels' => '@SyliusShop/Account/Order/Label/State',
+                        ],
+                    ])
+            )
+            ->addActionGroup(
+                ItemActionGroup::create(
+                    ShowAction::create([
+                        'link' => [
+                            'route' => 'sylius_shop_account_order_show',
+                            'parameters' => [
+                                'number' => 'resource.number',
+                            ],
+                        ],
+                    ]),
+                    Action::create('pay', 'shop_pay')
+                        ->setOptions([
+                            'link' => [
+                                'route' => 'sylius_shop_order_show',
+                                'parameters' => [
+                                    'tokenValue' => 'resource.tokenvalue',
+                                ],
+                            ],
+                        ])
```

Comment:
Missing the label, might also be the reason why behat fails (there's an `action.label` call in the template, I don't know if that property defaults in any way on the action's side).

Reply:
Yeah, that's probably the problem, didn't spot that.